actors/miner/src/lib.rs

```diff
@@ -878,6 +884,11 @@ impl Actor {
         }
         burn_funds(rt, aggregate_fee)?;
         state.check_balance_invariants(&rt.current_balance()).map_err(balance_invariants_broken)?;
+
+        for sector in proven_sector_nums {
+            emit::sector_proven(rt, SectorID { miner: miner_actor_id, number: sector })?;
+        }
```

Comment:
Nit:I suggest moving this up right under activate_new_sector_infos

Reply:
Done.